### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by Lawmsangi**
1.  I add a contact section.
2.  I style the contact section according to the Figma design.
3. I implemented form action.
4. The form was sent successfully to Formspree.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -219,6 +220,36 @@ <h3>Skills</h3>
...
+
+        <section class="contact-me" id="contact-me">
+            <div class="contact-container">
+                <div class="contact-content">
+                    <h2>Contact me</h2>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs
+                    coding. I’d love to help with it
+                    </p>
+                    <form action="/" method="post">
+                        <ul>
```
Comment (osamaashraf6) [inline index.html]: - [ ] Nice work implementing the form, it looks really good 👏 However it states in the project requirement that data should be collected by using the [Formspree service](https://formspree.io/html). Therefore I suggest you:😊⤵️
- 1. Add the action attribute to your form tag and its value should be similar to https://formspree.io/f/{form_id} where {form_id} is the id given you when creating an account on [formspree service](https://formspree.io/html).
- 2. Add the method attribute to your form tag and its value equals POST

- [ ] Hint: Click [here](https://youtu.be/sVPlb5C3oQ8) to watch this video to help you how to integrate the form spree with your form 😊💯

File: index.html [hunk 2]
```diff
@@ -219,6 +220,36 @@ <h3>Skills</h3>
...
+                            <li>
+                                <input type="text" id="name" placeholder="Enter your name">
+                            </li>
```
Comment (osamaashraf6) [inline index.html]: - [ ] Kindly 😃 It required that the name field accepts up to 30 chars, therefore kindly add the maxlength="30" attribute to the name field.💯👌
 - [ ] Also Please 😊It required that the user shouldn't be able to submit a form when one or more fields are empty, Therefore kindly add the required attribute to the name fields.💯🚀

File: index.html [hunk 3]
```diff
@@ -219,6 +220,36 @@ <h3>Skills</h3>
...
+                            <li>
+                                <input type="email" id="email" placeholder="Enter your email address">
+                            </li>
```
Comment (osamaashraf6) [inline index.html]: - [ ] Please 😊It required that the user shouldn't be able to submit a form when one or more fields are empty, Therefore kindly add the required attribute to the name fields.💯🚀

File: index.html [hunk 4]
```diff
@@ -219,6 +220,36 @@ <h3>Skills</h3>
...
+                            <li>
+                                <textarea name="" id="textarea" placeholder="Write your message here..."></textarea>
+                            </li>
+
```
Comment (osamaashraf6) [inline index.html]: - [ ] Kindly 😃 It required that the textarea field accepts up to 500 chars, therefore kindly add the maxlength="500" attribute to the name field to stick to the requirements .💯👌
 - [ ] Also Please 😊It required that the user shouldn't be able to submit a form when one or more fields are empty, Therefore kindly add the required attribute to the name fields.💯🚀

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @Lawmsangi 😊, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
SoCloseAlmostGIF (3)

## Highlights🎉🎉

- [ ] Following the Figma design for the project😊👌
- [ ] Use the semantics elements💯
- [ ] Nice design 🚀

## Required Changes ♻️

- [ ] Please 😊 consider using best practices when you write commit messages, they should be capitalized sentences in the imperative form, for example: "Fix bug at CSS file" instead of " fix or fixed Bug in CSS file"  ... etc 👍
_Check the comments under the review._

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — APPROVED** [review]
Hi @Lawmsangi , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] Profesional README ✔️ 
- [ ] All linter checks are passing ✔️ 
- [ ] Form validation has been implemented ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**